components/inputs/input-text.js

```diff
@@ -482,8 +482,8 @@ class InputText extends InputInlineHelpMixin(PropertyRequiredMixin(FocusMixin(La
 		let tooltip = nothing;
 		if (!this.skeleton) {
 			if (this.validationError && !this.noValidate) {
-				// this tooltip is using "announced" since we don't want aria-describedby wire-up which would bury the message in VoiceOver's More Content Available menu
-				tooltip = html`<d2l-tooltip state="error" announced align="start" class="vdiff-target">${this.validationError} <span class="d2l-offscreen">${this.description}</span></d2l-tooltip>`;
+				tooltip = html`<d2l-tooltip state="error" announced for="${this._inputId}" align="start" class="vdiff-target">${this.validationError} <span class="d2l-offscreen">${this.description}</span></d2l-tooltip>`;
```

Comment:
I _think_ we want this to be:
```html
<d2l-tooltip state="error" for="${this._inputId}" align="start" class="vdiff-target">${this.validationError}</d2l-tooltip>
```

* By using `for` and pointing at the `input` element, we should get the desired `aria-describedby` wire-up (by `d2l-tooltip`), making `announced` unnecessary

* The `input` will potentially already have `aria-describedby` if the consumer provided a `description` or content in the `inline-help` slot (see line 432 above). The `d2l-tooltip`'s `for` logic will append the tooltip id to those [here](https://github.com/BrightspaceUI/core/blob/main/components/tooltip/tooltip.js#L944). So, the offscreen description inside the original tooltip should no longer be necessary. I suspect that might have been rendered to handle the case where the `announce(...)` helper could clobber the normal `aria-describedby` announcement - that would no longer be an issue.

* Previously, the announcement order was: validation message, then the offscreen description, omitting the inline-help. If we still want to omit the inline-help, then we'll need another adjustment above on line 432 if there is a validation error. Also note: the helper appends the id to the end of `aria-describedby` [here in elemIdListAdd](https://github.com/BrightspaceUI/core/blob/main/helpers/dom.js#L25). The order of the ids in `aria-describedby` might dictate the order those messages are announced. I think I checked that previously but I don't totally recall what I found. If so, and if we still want the validation message announced first, we might need to add an option to that helper so the caller can indicate whether the id should be appended to the beginning or end.